### DISCUSSION TITLE
md生成的文件如果不带.html扩展名，会导致nginx中的mine.types文件配置无法匹配，会默认default_type，自有服务器可以更改成text/html，但是如果发布到github pages里面就会出现刷新页面，下载该页面的

### DISCUSSION THREAD

**Discussion by heyanming** [hexojs/hexo#5447] [Question / Answer] [answered]
### Check List

- [ ] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting).
- [ ] I have already searched existing issues and they are not help to me.
- [ ] I examined error or warning messages and it's difficult to solve.
- [ ] I am using the [latest](https://github.com/hexojs/hexo/releases) version of Hexo. (run `hexo version` to check)
- [ ] My Node.js is matched [the required version](https://hexo.io/docs/#Required-Node-js-version).

### Expected behavior

如果生成不带扩展名的文件也能兼容github page，不会出现刷新某个文章自动下载的情况

### Actual behavior

如果生成不带扩展名的文件也能兼容github page，出现刷新某个文章自动下载的情况

### How to reproduce?

见下图
![1711685621598](https://github.com/hexojs/hexo/assets/19993635/ec9bb938-b586-42a5-9dfb-d67d45c80b10)

### Is the problem still there under `Safe mode`?

11

**Comment by uiolee** [comment, marked as answer]
This depends on the server, and has nothing to do with hexo
